docs/api/iiasa.rst

```diff
@@ -1,15 +1,48 @@
 .. currentmodule:: pyam.iiasa
 
-The **Connection** class
-========================
+Databases hosted by IIASA
+=========================
 
-IIASA's ixmp Scenario Explorer infrastructure implements a RestAPI
-to directly query the database server connected to an explorer instance.
-See https://software.ene.iiasa.ac.at/ixmp-server for more information.
+The |pyam| package allows to directly query the scenario databases hosted by the
+IIASA Energy, Climate and Environment program (ECE), commonly known as
+the *Scenario Explorer* infrastructure. It is developed and maintained
+by the ECE `Scenario Services and Scientific Software team`_.
+
+.. _`Scenario Services and Scientific Software team` : https://software.ece.iiasa.ac.at
+
+You do not have to provide username/password credentials to connect to any public
+database instance using |pyam|. However, to connect to project-internal databases,
+you have to create an account at the IIASA-ECE *Manager Service*
+(https://manager.ece.iiasa.ac.at). Please contact the respective project coordinator
+for permission to access a project-internal database.
+
+To store the credentials on your machine so that |pyam| can use it to query a database,
+we depend on the Python package |ixmp4|. You only have to do this once
+(unless you change your password).
+
+In a console, run the following:
+
+.. code-block::
+
+    ixmp4 login <username>
+
+You will be prompted to enter your password.
+
```

Comment:
Maybe we could put the information here that the credentials will still work on previous Scenario Explorers:
"""
Note that if you have previously used credentials to access project-internal databases, they are the same and access to older databases will still work.
"""

Reply:
Good point, added a sentence to that effect.